contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,968 @@
+
+Some would have you believe that autocapture generates "too much data" that's hard to deal with. Our answer?
+
+To quote Tony Stark: “An intelligence agency which fears intelligence is, historically, not awesome.”
+
+### Client libraries
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Mixpanel</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>JavaScript</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Recommended for full PostHog functionality</td>
+        </tr>
+        <tr>
+            <td><strong>React Native</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>Flutter</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Event tracking and user identification only</td>
+        </tr>
+        <tr>
+            <td><strong>iOS (Swift)</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Autocapture and session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>iOS (Objective-C)</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Autocapture and session recording not available</td>
+        </tr>
+        <tr>
+            <td><strong>Android</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>PostHog: Event tracking and user identification only</td>
+        </tr>
+        <tr>
+            <td><strong>Unity</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
```

Comment:
```suggestion
            <td className="text-center"><span className="text-red text-lg">✖</span></td>
```
Just because we can connect to unity doesn't mean we have a client library for them (in my opinion)

Reply:
Ah, this is an error, should be the other way around. Mixpanel supports it, we don't.